cfme/tests/configure/test_docs.py

```diff
@@ -67,6 +67,20 @@ def docs_info():
             'User',
             'Settings and Operations'
         ]
+    elif version.appliance_is_downstream():
+        return [
+            'Monitoring Alerts Reporting',
+            'General Configuration',
+            'Virtual Machines Hosts',
+            'Methods For Automation',
+            'Infrastructure Inventory',
+            'Providers',
+            'Scripting Actions',
+            'Defining Policies Profiles'
+        ]
+    else:
+        # Upstream version has no docs
+        return []
```

Comment:
I dont believe we should be doing this. We should rather mark all tests to skip on upstream. The current solution will give us meaningless PASSes and will cost us some time as well, as it would navigate and then do nothing.